packages/rocketchat-livechat/server/models/LivechatInquiry.js

```diff
@@ -0,0 +1,73 @@
+class LivechatInquiry extends RocketChat.models._Base {
+	constructor() {
+		super();
+		this._initModel('livechat_inquiry');
+
+		this.tryEnsureIndex({ 'rid': 1 }); // room id corresponding to this inquiry 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2726404280/issues/source?bid=3199634&fileBranchId=3348698#l6)